### PR TITLE
Add recipe for avrocpp

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@conda-forge-admin Any hints for the py27 windows builds?  Should they even be made?

**Comment by CJ-Wright** [comment]
I have no idea about the windows build, but I think the `admin` is a bot, you want @conda-forge/staged-recipes (the review team for staged-recipes)

**Comment by mariusvniekerk** [comment]
@conda-forge/staged-recipes Any hints for the py27 windows builds? Should they even be made?

**Comment by isuruf** [comment]
@mariusvniekerk, if you want, you can disable them. Or you can patch `buffertest.cc` and add `#include <stdint.h>` to the top

File: recipes/avrocpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    version: {{ version }}
+
+source:
+    fn: avro-cpp-{{ version }}.tar.gz
+    url: https://archive.apache.org/dist/avro/avro-{{ version }}/cpp/avro-cpp-{{ version }}.tar.gz
+    sha1: {{ hash }}
+
+build:
+    number: 0
+    skip: true  # [win]
```
Comment (dougalsutherland) [inline recipes/avrocpp/meta.yaml]: It seemed like only `win and py2k` didn't work, but now you're skipping all Windows builds? Is there a reason not to do the newer-platform Windows builds?

If you do, you should probably add <code>python &nbsp;# [win]</code> as a `build` requirement, add [VC features](https://conda-forge.org/docs/buildwin.html#vc-features) to the `build` block, and change it to <code>skip: true &nbsp;# [win and not py36]</code>.

File: recipes/avrocpp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+    commands:
+        - test -f ${PREFIX}/lib/libavrocpp_s.a  # [unix]
+        - conda inspect objects {{ name }}      # [osx]
+        - conda inspect linkages {{ name }}     # [linux]
+
+about:
+  home: http://hadoop.apache.org/avro
+  license: Apache 2.0
```
Comment (dougalsutherland) [inline recipes/avrocpp/meta.yaml]: Should also add `license_family: Apache` and `license_file: LICENSE`.

Comment (mariusvniekerk) [reply recipes/avrocpp/meta.yaml]: Done 🏁

**Comment by scopatz** [comment]
Thanks @mariusvniekerk!

**Pull request merged**